server/routes/__tests__/updates.spec.js

```diff
@@ -0,0 +1,251 @@
+const request = require('supertest');
+const cheerio = require('cheerio');
+
+jest.mock('@sentry/node');
+
+const { createUpdatesContentRouter } = require('../updates');
+const { setupBasicApp } = require('../../../test/test-helpers');
+
+describe('GET /updates', () => {
+  let app;
+
+  const cmsService = {
+    getUpdatesContent: jest.fn(),
+  };
+
+  const sessionMiddleware = (req, res, next) => {
+    req.session = {
+      establishmentId: 123,
+      establishmentName: 'berwyn',
+    };
+    next();
+  };
+
+  let relatedContent;
+  let data;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    app = setupBasicApp();
+    app.use(sessionMiddleware);
+    app.use('/', createUpdatesContentRouter({ cmsService }));
+
+    relatedContent = [
```

Comment:
relatedContent doesn't appear to be modified. This should be a const in describe and not within the beforeEach?

Reply:
Objects and Arrays contents are not immutable and therefore increase the risk of test bleed.